src/js/pages/HowItWorks.jsx

```diff
@@ -256,6 +258,7 @@ class HowItWorks extends Component {
   }
 
   handleChangeSlide = (selectedStepIndex) => {
+    this.sendHowItWorksSlideEvent(selectedStepIndex);
```

Comment:
There needs to be a function that gets the titleId from the selectedStepIndex, so we can pass titleId as the buttonId in actionDetails.